src/index.html

```diff
@@ -8,10 +8,173 @@
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
+
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>HTML Form</h1>
+    <form action="http://127.0.0.1:5500/src/index.html"
+    method="post"
+    class="form_basic"
+    autocomplete="off">
+      <fieldset class="field_inform"> <!--Form information-->
+        <legend class="form_title_information">Personal information</legend>
+        <label class="surname">
+          Surname:
+          <input type="text"
+          name="surname"
+          class="surname_input"
+          required
+          >
```

Comment:
Pay attention on indents. Fix all similar places
```suggestion
          <input 
            type="text"
            name="surname"
            class="surname_input"
            required
          >
```